awx/main/management/commands/inventory_import.py

```diff
@@ -577,16 +595,29 @@ def _delete_group_children_and_hosts(self):
             for offset in range(0, len(del_host_pks), self._batch_size):
                 del_pks = del_host_pks[offset:(offset + self._batch_size)]
                 for db_host in db_hosts.filter(pk__in=del_pks):
-                    group_host_count += 1
```

Comment:
Note that `group_host_count` here meant something like the count of group-host memberships that overlapped between the imported content and database content.

It did not mean the number of relationships removed.